bigquery/google/cloud/bigquery/job.py

```diff
@@ -981,10 +1072,12 @@ def max_bad_records(self):
         See
         https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.maxBadRecords
         """
+        # XXX: should this coerce from str, like 'skip_leading_rows' below?
```

Comment:
It's marked as int32 in the discovery document, so it should come over the wire as an integer. It wouldn't hurt to add an extra `int` call in case that ever changes.